core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerPool.scala

```diff
@@ -271,6 +336,13 @@ class ContainerPool(childFactory: ActorRefFactory => ActorRef,
       //backfill prewarms on every ContainerRemoved, just in case
       backfillPrewarms(false) //in case a prewarm is removed due to health failure or crash
 
+    // prewarmed container got removed
+    case PreWarmedContainerRemoved =>
+      prewarmedPool.get(sender()).foreach { _ =>
+        logging.info(this, "prewarmed container is deleted due to unused for long time")
```

Comment:
Would be great to include the type of prewarm container.

Reply:
Already added